test/functional/terminal/ex_terminal_spec.lua

```diff
@@ -90,6 +102,18 @@ describe(':terminal (with fake shell)', function()
     ]])
   end)
 
+  it('executes a given command through the shell, when the shell has arguments', function()
```

Comment:
it("executes a given command through the shell, when 'shell' has arguments', function()